docs/Tutorial.md

```diff
@@ -515,8 +518,79 @@ const App = () => (
     </Admin>
 );
 ```
+The post list now displays usernames on each line. To add a deep link to display a detailed view of that user you will need to follow these steps:
```

Comment:
I think this section arrives too early, and is too large to fit in an existing section. Or, to put it otherwise, it deserves its own section, but later in the tutorial.

Reply:
I ended up adding a new section with more details.